GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -255,7 +255,7 @@ private void DehydrateFolders(JsonTracer tracer, GVFSEnlistment enlistment, stri
                                 }
                                 else
                                 {
-                                    string fullPath = Path.Combine(enlistment.WorkingDirectoryRoot, folder);
+                                    string fullPath = Path.Combine(enlistment.WorkingDirectoryBackingRoot, folder);

```

Comment:
Why is this change important?

Reply:
On Windows and Mac `WorkingDirectoryRoot == WorkingDirectoryBackingRoot` but on Linux that is not the case and when working with the unmounted repo it is in the `WorkingDirectoryBackingRoot`.  So this really is a fix for dehydrate folders to make sure it will work on Linux.

Reply:
Ah. I didn't realize we are unmounted at this point. Thanks!